octoprint_PrintTimeGenius/analyzers/analyze_gcode_comments.py

```diff
@@ -6,10 +6,12 @@
 import sys
 from collections import defaultdict
 import argparse
-if float('.'.join((str(sys.version_info.major), str(sys.version_info.minor)))) < 3.3:
-  from collections import Mapping as collections_Mapping
+
+python_version_above_3_3 = [sys.version_info.major, sys.version_info.minor] > [3, 3]
```

Comment:
`sys.version_info` is a named tuple as explained here: https://docs.python.org/3/library/sys.html#sys.version_info

So you can write:

```python
python_version_above_3_3 = sys.version_info > (3,3)
```

I also prefer to consistently have all indentation be just two spaces per level, as the file is already.

Thanks for your dilegence!

Reply:
Yeah I double-checked this with pyenv:

```
$ for V in 3.2.5   3.3.0   3.3.5   3.4.4   3.6.8   3.9.12   3.9.13   3.10.10   3.11.2; do export PYENV_VERSION="$V"; python -c 'import sys; python_version_above_3_3 = sys.version_info > (3, 3); print("Python Version: ", sys.version_info); print("Python Version is above
3.3: ", python_version_above_3_3);'; echo -e "=======\n"; done
Python Version:  sys.version_info(major=3, minor=2, micro=5, releaselevel='final', serial=0)
Python Version is above 3.3:  False
=======

Python Version:  sys.version_info(major=3, minor=3, micro=0, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=3, micro=5, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=4, micro=4, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=6, micro=8, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=9, micro=12, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=9, micro=13, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=10, micro=10, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======

Python Version:  sys.version_info(major=3, minor=11, micro=2, releaselevel='final', serial=0)
Python Version is above 3.3:  True
=======
```